.github/workflows/upgrade-python-requirements.yml

```
@@ -0,0 +1,30 @@
+name: Upgrade Python Requirements
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"
```

```suggestion
    # https://crontab.guru/#0_0_*_*_1
    # At midnight on Mondays
    - cron: "0 0 * * 1"
```
It's always nice to have a comment about what the Cron is meant to be since not everyone naturally reads cron syntax.